pyproject.toml

```
@@ -50,6 +51,9 @@ Repository = "https://github.com/pybamm-team/PyBaMM"
 Releases = "https://github.com/pybamm-team/PyBaMM/releases"
 Changelog = "https://github.com/pybamm-team/PyBaMM/blob/develop/CHANGELOG.md"
 
+[tool.hatch.version]
+source = "vcs"
```

Can we move this down to the already existing `tool.hatch` section?
```suggestion
[tool.hatch]
version.source = "vcs"
build.hooks.vcs.version-file = "src/pybamm/_version.py"
```

---

Sure, I'll move it to the `tool.hatch` section.